jsaddle/jsaddle.cabal

```diff
@@ -46,7 +46,7 @@ library
             ghc-prim,
             http-types >=0.8.6 && <0.13,
             process >=1.2.3.0 && <1.7,
-            random >= 1.1 && < 1.3,
+            random >= 1.1 && < 1.4,
```

Comment:
`random-1.3` is only picked up with `--allow-newer=random` because of  the `uuid` dependency, but passes `cabal test all`.